src/pages/new/docs/configuration/scheduling/index.vue

```diff
@@ -0,0 +1,85 @@
+    </v-col>
+  </v-row>
+</template>
+
+<script setup lang="ts">
+
+import { marked } from 'marked'
+
+const markdown1 = marked(`
+# Scheduling
+---
+
+For scheduling any recurring jobs, NEXUSe2e uses a library called Quartz.\\
+It is used to regulate how many jobs are executed at the same time, how often different jobs run and what happens, if a job fails to run (misfires).
+
+Jobs include creating and sending messages, saving incoming messages, logging, etc.\\
+Most job scheduling in NEXUSe2e is done with preconfigured values that do not require any explicit configuration to run.\\
+Some user configured services like the [DirectoryScannerService](#/new/docs/components/services/directoryScanner) require explicit scheduling configuration however.
+
+## Number of parallel jobs
+
+By default, NEXUSe2e allows 5 jobs to be executed at the same time (5 quartz threads - default number of threads is subject to change), but this can configured in the NEXUSe2e application properties.\\
+For NEXUSe2e running in Tomcat the application properties are usually located in the application-nexuse2e.yml in the Tomcat /conf directory.
+It is important to keep in mind that more threads/parallel jobs might require more system resources (memory).
+
+Example configuration (to allow 10 parallel jobs):
+`)
+
+const markdown2 = marked(`
+    spring:
+      quartz:
+        properties:
+          org.quartz.threadPool.threadCount: 10
+`)
+
+const markdown3 = marked(`
+## Intervals and the Misfire Threshold
+
+The misfire threshold determines how much time is allowed until NEXUSe2e consideres a job to have misfired.\\
+If, for example, a job is scheduled to run once at 12:00am and the misfire threshold is set to 5 minutes, NEXUSe2e will consider a job to have run on time, if it is executed before 12:05am.\\
+If the job is not executed before 12:05am, it is treated as a misfire.\\
+How misfires are treated depends on the misfire instruction (usually "next schedule", meaning the execution of the job is skipped until the next scheduled time).
+
+There can be several reason a job may not executed on time (misfires):
+- The application (NEXUSe2e) was not running at the time.
+- The service was not running at the time.
+- Too many jobs where queued for execution and alle threads were busy.
+
+A single misfire is usually no issue, especially with low intervals (e.g. 5 minutes).\\
+With the "next schedule" misfire instruction the job execution would just be skipped and the job would just be executed 5 minutes later.\\
+If jobs are not getting executed in the desired amount of time though, adjustments might have to be made to the scheduling settings.
+
+For longer intervals (e.g. daily) in combination with the "next schedule" misfire instruction a single misfire would result in the job not running that day at all though.\\
+In that case an alternative misfire Instruction (e.g. "run once") might be better.\\
+What misfire instruction options are available depends on the service.
```

Comment:
warum hängt das vom Service ab? dachte, wenn man es angeben kann, sind es immer die beiden?

Reply:
Theoretisch können die Optionen in jeder Service Komponente unterschiedlich sein, solang man nicht die Default definition aus dem Interface nutzt.